package.json

```diff
@@ -2,6 +2,7 @@
   "name": "math-magicians",
```

Comment:
Well done! You have set up the environment and tools needed to develop the React application ✔ App is running as expected 🤓